projects/Spell_checker/spell_checker.py

```diff
@@ -0,0 +1,12 @@
+from textblob import TextBlob    # importing textblob library
+
+t = 1
+while t:
+    a = input("Enter the word to be checked:- ")	 # incorrect spelling
+    print("original text: "+str(a))     #printing original text
```

Comment:
Instead of asking for a word to be checked, ask for an input string;
Now, split the string, to form a list of words;

```python
words = list(a.split())
```